docs/api/checkouts.rst

```diff
@@ -1,235 +1,560 @@
-        "updated_at": "2014-12-15T13:37:29+0000",
-        "user": {
-            "credentials_expired": false,
-            "email": "xschaefer@example.com",
-            "email_canonical": "xschaefer@example.com",
-            "enabled": true,
-            "expired": false,
-            "group": [],
-            "id": 5,
-            "locked": false,
-            "roles": [],
-            "username": "xschaefer@example.com",
-            "username_canonical": "xschaefer@example.com"
-        }
+        "billing_address":{
+            "first_name":"Elon",
+            "last_name":"Musk",
+            "country_code":"US",
+            "street":"10941 Savona Rd",
+            "city":"\u2019Los Angeles",
+            "postcode":"CA 90077"
+        },
+        "payments":[
+            {
+                "id":21,
+                "method":{
+                    "id":1,
+                    "code":"cash_on_delivery"
+                },
+                "amount":108787,
+                "state":"cart"
+            }
+        ],
+        "shipments":[
+            {
+                "id":21,
+                "state":"cart",
+                "method":{
+                    "code":"ups",
+                    "enabled":true
+                }
+            }
+        ],
+        "currency_code":"USD",
+        "locale_code":"en_US",
+        "checkout_state":"addressed"
     }
 
+Of course you can specify different shipping and billing address. If Elon would like to to send a present to NASA administrator, Frederick D. Gregory, he could send the following request:
 
-Shipping step
--------------
+.. code-block:: bash
 
-When order contains the address information, we are able to determine the stock locations and available shipping methods.
-You can get these informations by first calling a GET request on the checkout unique URL.
+    $ curl http://127.0.0.1:8000/api/v1/checkouts/addressing/21 \
+        -H "Authorization: Bearer SampleToken" \
+        -H "Content-Type: application/json" \
+        -X PUT \
+        --data '
+            {
+                "shipping_address": {
+                    "first_name": " Frederick D.",
+                    "last_name": "Gregory",
+                    "street": "300 E St SW",
+                    "country_code": "US",
+                    "city": "’Washington",
+                    "postcode": "DC 20546"
+                },
+                "different_billing_address": false,
+                "billing_address": {
+                    "first_name": "Elon",
+                    "last_name": "Musk",
+                    "street": "10941 Savona Rd",
+                    "country_code": "US",
+                    "city": "’Los Angeles",
+                    "postcode": "CA 90077"
+                }
+            }
+        '
+
+Exemplary Response
+^^^^^^^^^^^^^^^^^^
 
 .. code-block:: text
 
-    GET /api/v1/checkouts/44
+    STATUS: 204 No Content
+
+Now you can check the state of order asking for checkout summary:
```

Comment:
`Now you can check the state of order asking for checkout summary:` I actually do not like this sentence. Especially when you read it more than once in the same article. Is it needed here? 

And the below example: **It's duplication actually.**